rasa/core/actions/loops.py

```diff
@@ -43,6 +44,7 @@ async def is_activated(
 
     # default implementation checks if form active
     def _default_activation_events(self) -> List[Event]:
+        # TODO if this is in the loop action, probably it should not be `Form`
```

Comment:
You mean we should rename the event?

Reply:
yes, and also think whether we want to use the same event name for forms and loops. So we need to think through how they interact with each other. Let's tackle it separately. I added `TODO` so that we don't forget